tokio-timer/src/error.rs

```diff
@@ -0,0 +1,39 @@
+use self::Kind::*;
+
+/// Errors encountered by the timer implementation.
+#[derive(Debug)]
+pub struct Error(Kind);
```

Comment:
Once again, `std::error::Error` impls will probably be good to have...

Reply:
You have an unnamed field. Compare to:

```
pub struct DeadlineError<T> {
    kind: Kind<T>,
}
```

Reply:
I switched to tuple structs for consistency.